app/javascript/modules/graph/index.js

```diff
@@ -0,0 +1,16 @@
+const handleSidebarChange = () => {
+  let selectInput = document.querySelector('select.period-selection');
+  let optionSelected = selectInput.options[selectInput.selectedIndex].text;
+  const sidebarForm = document.querySelector('.sidebar-form');
+
+  if (optionSelected === '') {
+    const url = window.location.href.split('?')[0]
+    window.history.replaceState({}, document.title, url);
+  }
```

Comment:
for a future PR. I think this code and 
```js
  let selectInput = document.querySelector('select.period-selection');
  let optionSelected = selectInput.options[selectInput.selectedIndex].text;

  if (optionSelected === '') {
    const url = window.location.href.split('?')[0]
    window.history.replaceState({}, document.title, url);
  }
```
should be moved to a different file because it has nothing to do with handling the sidebar change, right? It's just something we want to run once the page is loaded

Reply:
you are right, i will change it